src/chrome/content/rules/ENS.fr.xml

```diff
@@ -110,28 +117,42 @@
 
 	<!--	Direct rewrites:
 				-->
-	<target host="www.astreea.ens.fr" />
 	<!--target host="astro.ens.fr" /-->
+
+	<target host="www.ens.fr" />
+
 	<target host="www.biologie.ens.fr" />
+	<target host="www.actarus.biologie.ens.fr" />
+	<target host="cloud.biologie.ens.fr" />
 	<target host="www-mail.biologie.ens.fr" />
+	<target host="zebrain.biologie.ens.fr" />
+	<target host="www.zebrain.biologie.ens.fr" />
+	
+	<target host="challengedata.ens.fr" />
 	<target host="www.cof.ens.fr" />
 
 	<target host="crypto.di.ens.fr" />
 	<target host="cryptobib.di.ens.fr" />
 	<target host="www.di.ens.fr" />
-
 	<target host="dematec.ens.fr" />
 	<target host="www.dg.ens.fr" />
+	<target host="www.documents.ens.fr" />
+	<target host="ecocampus.ens.fr" />
 	<target host="www.ecocampus.ens.fr" />
 	<target host="git.eleves.ens.fr" />
 	<target host="www.eleves.ens.fr" />
 	<target host="hack.ens.fr" />
 	<target host="halley.ens.fr" />
+	
+		<!-- Used to access papers via google scholar. Causes SSL_ERROR_RX_RECORD_TOO_LONG -->
+		<exclusion pattern="^http://halley\.ens\.fr:4550/" />
```

Comment:
You can use `http://halley.ens.fr:4550/robots.txt` as a test url to remove it from the whitelist.